docs/en/ingest-management/elastic-agent/configuring-kubernetes-metadata.asciidoc

```diff
@@ -0,0 +1,120 @@
+[[configuring-kubernetes-metadata]]
+= Configuring Kubernetes metadata enrichment on {agent}
+
+Kubernetes https://www.elastic.co/guide/en/observability/current/monitor-kubernetes.html#beats-metadata[metadata] refer to contextual information extracted from Kubernetes resources. Metadata information enrich metrics and logs
+collected from a Kubernetes cluster, enabling deeper insights in Kubernetes environments.
+
+When the elastic-agent's policy includes https://docs.elastic.co/en/integrations/kubernetes[{k8s} Integration] which configures the collection of Kubernetes related metrics and container logs, the mechanisms used for the metadata enrichment are:
```

Comment:
```suggestion
When the {agent}'s policy includes the https://docs.elastic.co/en/integrations/kubernetes[{k8s} Integration] which configures the collection of Kubernetes related metrics and container logs, the mechanisms used for the metadata enrichment are:
```
We have a couple of ahndy variables we can use for agent:
`{agents}` = Elastic Agents
`{agent}` = Elastic Agent